apps/financial-remedy/finrem-cos/demo.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     java:
       disableIngressClassAnnotation: true
       ingressClass: traefik-private
-      image: hmctspublic.azurecr.io/finrem/cos:pr-894-5d2e021-20221223170235 #{"$imagepolicy": "flux-system:demo-finrem-cos"}
```

Comment:
I don't think you need to make any change here

Reply:
There are two way to change to prod image, either we can change to flux-system:demo-finrem-cos to flux-system:finrem-cos OR change in demo-image-policy.yaml in file